### DISCUSSION TITLE
[Question] Database template variables and locals

### DISCUSSION THREAD

**Discussion by XenonR** [azerothcore/azerothcore-wotlk#11602] [General]
Hi very good work. I would like to add some locals to the DB and was wondering about a few things:

1. Is it more preferable to use \n or $B for new lines?
2. Shouldn't "newlines" be harmonized across the DB?
3. And if I add new rows to the SQL is it preferred to append them or "sort them in" by ID?

Thanks.

Edit: I read the [SQL Standards](https://www.azerothcore.org/wiki/sql-standards)

**Comment by Kitzunu** [comment]
1. $B
2. Newlines are always required
3. append them to a certain degree. f.ex don't append 100 rows of SAI for different creatures, there it is preffered to have one query per entry + actionlist
